doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -307,6 +307,8 @@ Rewriting with Leibniz and setoid equality
 
    .. flag:: Regular Subst Tactic
 
+      .. XXX this flag does not exist anymore, is this simple subst?
```

Comment:
looking at 3ddcc48467b8605c17b78df9cc6220d4189b3aa3 this flag was not simple subst